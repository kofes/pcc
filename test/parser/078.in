function foo(var v:integer):integer;
begin
end;
procedure foo;
begin
end;

begin
end
