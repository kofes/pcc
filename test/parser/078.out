<function table>
   <procedure><name>FOO</name>
      <body>
      </body>
   </procedure>
   <function><name>FOO</name>
      <return type>INTEGER</return type>
      <params>
         <param>V:v:INTEGER</param>
      </params>
      <body>
      </body>
   </function>
</function table>
<type table>
   <type>BOOLEAN</type>
   <type>CHAR</type>
   <type>INTEGER</type>
   <type>REAL</type>
</type table>
<var table>
   <var>PI:C:REAL:3.1415926535897932385</var>
</var table>
<block statement>
</block statement>
